From e7be39cb841bb12a664a7a60e983b7060c39ed5a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Tue, 31 Mar 2015 13:44:27 -0700
Subject: [PATCH 5/9] Cleans up empty comment lines

They clutter the code.
It also attempts to achieve coherent spacing (2 lines) between functions.
---
 xdgaction.cpp               |  33 -----------
 xdgdesktopfile.cpp          | 138 +++-----------------------------------------
 xdgdesktopfile.h            |   1 -
 xdgdirs.cpp                 |  32 ++--------
 xdgdirs.h                   |   2 -
 xdgicon.cpp                 |  20 -------
 xdgicon.h                   |   2 -
 xdgmenu.cpp                 |  84 +--------------------------
 xdgmenu.h                   |   3 -
 xdgmenu_p.h                 |   2 -
 xdgmenuapplinkprocessor.cpp |  22 -------
 xdgmenuapplinkprocessor.h   |   3 +-
 xdgmenulayoutprocessor.cpp  |  19 ------
 xdgmenulayoutprocessor.h    |   1 -
 xdgmenureader.cpp           |  18 ------
 xdgmenureader.h             |   2 +-
 xdgmenurules.cpp            |  54 -----------------
 xdgmenurules.h              |   1 +
 xdgmenuwidget.cpp           |  33 +----------
 xdgmenuwidget.h             |   1 -
 xdgmimetype.cpp             |  10 ++++
 21 files changed, 28 insertions(+), 453 deletions(-)

diff --git a/xdgaction.cpp b/xdgaction.cpp
index 29c1712..9fd24fe 100644
--- a/xdgaction.cpp
+++ b/xdgaction.cpp
@@ -32,18 +32,12 @@
 #include <QCoreApplication>
 
 
-/************************************************
-
- ************************************************/
 XdgAction::XdgAction(QObject *parent):
     QAction(parent)
 {
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction::XdgAction(const XdgDesktopFile& desktopFile, QObject *parent):
     QAction(parent)
 {
@@ -51,9 +45,6 @@ XdgAction::XdgAction(const XdgDesktopFile& desktopFile, QObject *parent):
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction::XdgAction(const XdgDesktopFile* desktopFile, QObject *parent):
     QAction(parent)
 {
@@ -61,9 +52,6 @@ XdgAction::XdgAction(const XdgDesktopFile* desktopFile, QObject *parent):
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction::XdgAction(const QString& desktopFileName, QObject *parent):
     QAction(parent)
 {
@@ -73,9 +61,6 @@ XdgAction::XdgAction(const QString& desktopFileName, QObject *parent):
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction::XdgAction(const XdgAction& other, QObject *parent):
     QAction(parent)
 {
@@ -83,17 +68,11 @@ XdgAction::XdgAction(const XdgAction& other, QObject *parent):
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction::~XdgAction()
 {
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction& XdgAction::operator=(const XdgAction& other)
 {
     load(other.mDesktopFile);
@@ -101,18 +80,12 @@ XdgAction& XdgAction::operator=(const XdgAction& other)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgAction::isValid() const
 {
     return mDesktopFile.isValid();
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgAction::load(const XdgDesktopFile& desktopFile)
 {
     mDesktopFile = desktopFile;
@@ -134,9 +107,6 @@ void XdgAction::load(const XdgDesktopFile& desktopFile)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgAction::runConmmand() const
 {
     if (mDesktopFile.isValid())
@@ -144,9 +114,6 @@ void XdgAction::runConmmand() const
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgAction::updateIcon()
 {
     setIcon(mDesktopFile.icon());
diff --git a/xdgdesktopfile.cpp b/xdgdesktopfile.cpp
index 8905a88..c46a4a7 100644
--- a/xdgdesktopfile.cpp
+++ b/xdgdesktopfile.cpp
@@ -91,9 +91,8 @@ void replaceVar(QString &str, const QString &varName, const QString &after);
 QString &unEscape(QString& str);
 QString &unEscapeExec(QString& str);
 void loadMimeCacheDir(const QString& dirName, QHash<QString, QList<XdgDesktopFile*> > & cache);
-/************************************************
 
- ************************************************/
+
 QString &doEscape(QString& str, const QHash<QChar,QChar> &repl)
 {
     // First we replace slash.
@@ -159,10 +158,6 @@ QString &escapeExec(QString& str)
 }
 
 
-
-/************************************************
-
- ************************************************/
 QString &doUnEscape(QString& str, const QHash<QChar,QChar> &repl)
 {
     int n = 0;
@@ -202,7 +197,6 @@ QString &unEscape(QString& str)
 }
 
 
-
 /************************************************
  Quoting must be done by enclosing the argument between double quotes and
  escaping the
@@ -292,9 +286,6 @@ public:
 };
 
 
-/************************************************
-
- ************************************************/
 XdgDesktopFileData::XdgDesktopFileData():
     mIsValid(false),
     mValidIsChecked(false)
@@ -302,9 +293,6 @@ XdgDesktopFileData::XdgDesktopFileData():
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFileData::read(const QString &prefix)
 {
     QFile file(mFileName);
@@ -348,9 +336,6 @@ bool XdgDesktopFileData::read(const QString &prefix)
     return mIsValid;
 }
 
-/************************************************
-
- ************************************************/
 XdgDesktopFile::Type XdgDesktopFileData::detectType(XdgDesktopFile *q) const
 {
     QString typeStr = q->value("Type").toString();
@@ -370,9 +355,6 @@ XdgDesktopFile::Type XdgDesktopFileData::detectType(XdgDesktopFile *q) const
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFileData::startApplicationDetached(const XdgDesktopFile *q, const QStringList& urls) const
 {
     //DBusActivatable handling
@@ -429,9 +411,6 @@ bool XdgDesktopFileData::startApplicationDetached(const XdgDesktopFile *q, const
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFileData::startLinkDetached(const XdgDesktopFile *q) const
 {
     QString url = q->url();
@@ -486,27 +465,18 @@ bool XdgDesktopFileData::startByDBus(const QStringList& urls) const
 }
 
 
-
-/************************************************
-
- ************************************************/
 XdgDesktopFile::XdgDesktopFile():
     d(new XdgDesktopFileData)
 {
 }
 
 
-/************************************************
-
- ************************************************/
 XdgDesktopFile::XdgDesktopFile(const XdgDesktopFile& other):
     d(other.d)
 {
 }
 
-/************************************************
 
- ************************************************/
 XdgDesktopFile::XdgDesktopFile(Type type, const QString& name, const QString &value):
     d(new XdgDesktopFileData)
 {
@@ -532,17 +502,11 @@ XdgDesktopFile::XdgDesktopFile(Type type, const QString& name, const QString &va
 }
 
 
-/************************************************
-
- ************************************************/
 XdgDesktopFile::~XdgDesktopFile()
 {
 }
 
 
-/************************************************
-
- ************************************************/
 XdgDesktopFile& XdgDesktopFile::operator=(const XdgDesktopFile& other)
 {
     d = other.d;
@@ -550,18 +514,12 @@ XdgDesktopFile& XdgDesktopFile::operator=(const XdgDesktopFile& other)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::operator==(const XdgDesktopFile &other) const
 {
     return d->mItems == other.d->mItems;
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::load(const QString& fileName)
 {
     if (fileName.startsWith(QDir::separator())) { // absolute path
@@ -580,9 +538,6 @@ bool XdgDesktopFile::load(const QString& fileName)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::save(QIODevice *device) const
 {
     QTextStream stream(device);
@@ -606,9 +561,6 @@ bool XdgDesktopFile::save(QIODevice *device) const
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::save(const QString &fileName) const
 {
     QFile file(fileName);
@@ -619,9 +571,6 @@ bool XdgDesktopFile::save(const QString &fileName) const
 }
 
 
-/************************************************
-
- ************************************************/
 QVariant XdgDesktopFile::value(const QString& key, const QVariant& defaultValue) const
 {
     QString path = (!prefix().isEmpty()) ? prefix() + "/" + key : key;
@@ -636,9 +585,6 @@ QVariant XdgDesktopFile::value(const QString& key, const QVariant& defaultValue)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgDesktopFile::setValue(const QString &key, const QVariant &value)
 {
     QString path = (!prefix().isEmpty()) ? prefix() + "/" + key : key;
@@ -663,9 +609,6 @@ void XdgDesktopFile::setValue(const QString &key, const QVariant &value)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgDesktopFile::setLocalizedValue(const QString &key, const QVariant &value)
 {
     setValue(localizedKey(key), value);
@@ -747,9 +690,7 @@ QString XdgDesktopFile::localizedKey(const QString& key) const
     return key;
 }
 
-/************************************************
 
- ************************************************/
 QVariant XdgDesktopFile::localizedValue(const QString& key, const QVariant& defaultValue) const
 {
     return value(localizedKey(key), defaultValue);
@@ -774,9 +715,7 @@ QStringList XdgDesktopFile::categories() const
     return cats;
 }
 
-/************************************************
 
- ************************************************/
 void XdgDesktopFile::removeEntry(const QString& key)
 {
     QString path = (!prefix().isEmpty()) ? prefix() + "/" + key : key;
@@ -784,9 +723,6 @@ void XdgDesktopFile::removeEntry(const QString& key)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::contains(const QString& key) const
 {
     QString path = (!prefix().isEmpty()) ? prefix() + "/" + key : key;
@@ -794,27 +730,18 @@ bool XdgDesktopFile::contains(const QString& key) const
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::isValid() const
 {
     return d->mIsValid;
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDesktopFile::fileName() const
 {
     return d->mFileName;
 }
 
 
-/************************************************
-
- ************************************************/
 QIcon const XdgDesktopFile::icon(const QIcon& fallback) const
 {
     QIcon result = XdgIcon::fromTheme(value("Icon").toString(), fallback);
@@ -828,18 +755,12 @@ QIcon const XdgDesktopFile::icon(const QIcon& fallback) const
 }
 
 
-/************************************************
-
- ************************************************/
 QString const XdgDesktopFile::iconName() const
 {
     return value("Icon").toString();
 }
 
 
-/************************************************
-
- ************************************************/
 XdgDesktopFile::Type XdgDesktopFile::type() const
 {
     return d->mType;
@@ -884,9 +805,6 @@ bool XdgDesktopFile::startDetached(const QString& url) const
 }
 
 
-/************************************************
-
- ************************************************/
 static QStringList parseCombinedArgString(const QString &program)
 {
     QStringList args;
@@ -928,9 +846,6 @@ static QStringList parseCombinedArgString(const QString &program)
 }
 
 
-/************************************************
-
- ************************************************/
 void replaceVar(QString &str, const QString &varName, const QString &after)
 {
     str.replace(QRegExp(QString("\\$%1(?!\\w)").arg(varName)), after);
@@ -938,9 +853,6 @@ void replaceVar(QString &str, const QString &varName, const QString &after)
 }
 
 
-/************************************************
-
- ************************************************/
 QString expandEnvVariables(const QString str)
 {
     QString scheme = QUrl(str).scheme();
@@ -978,9 +890,6 @@ QString expandEnvVariables(const QString str)
 }
 
 
-/************************************************
-
- ************************************************/
 QStringList expandEnvVariables(const QStringList strs)
 {
     QStringList res;
@@ -991,9 +900,6 @@ QStringList expandEnvVariables(const QStringList strs)
 }
 
 
-/************************************************
-
- ************************************************/
 QStringList XdgDesktopFile::expandExecString(const QStringList& urls) const
 {
     if (d->mType != ApplicationType)
@@ -1108,10 +1014,6 @@ QStringList XdgDesktopFile::expandExecString(const QStringList& urls) const
 }
 
 
-
-/************************************************
- Check if the program is actually installed.
- ************************************************/
 bool checkTryExec(const QString& progName)
 {
     if (progName.startsWith(QDir::separator()))
@@ -1129,9 +1031,6 @@ bool checkTryExec(const QString& progName)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::isShow(const QString& environment) const
 {
     const QString env = environment.toUpper();
@@ -1153,6 +1052,7 @@ bool XdgDesktopFile::isShow(const QString& environment) const
     return true;
 }
 
+
 bool XdgDesktopFile::isShown(const QString &environment) const
 {
     const QString env = environment.toUpper();
@@ -1175,9 +1075,6 @@ bool XdgDesktopFile::isShown(const QString &environment) const
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDesktopFile::isApplicable(bool excludeHidden, const QString& environment) const
 {
     // Hidden should have been called Deleted. It means the user deleted
@@ -1211,6 +1108,7 @@ bool XdgDesktopFile::isApplicable(bool excludeHidden, const QString& environment
     return true;
 }
 
+
 bool XdgDesktopFile::isSuitable(bool excludeHidden, const QString &environment) const
 {
     // Hidden should have been called Deleted. It means the user deleted
@@ -1276,9 +1174,7 @@ bool XdgDesktopFile::isSuitable(bool excludeHidden, const QString &environment)
     return true;
 }
 
-/************************************************
 
- ************************************************/
 QString expandDynamicUrl(QString url)
 {
     foreach(QString line, QProcess::systemEnvironment())
@@ -1293,9 +1189,6 @@ QString expandDynamicUrl(QString url)
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDesktopFile::url() const
 {
     if (type() != LinkType)
@@ -1316,10 +1209,6 @@ QString XdgDesktopFile::url() const
 }
 
 
-
-/************************************************
-
- ************************************************/
 QString findDesktopFile(const QString& dirName, const QString& desktopName)
 {
     QDir dir(dirName);
@@ -1345,9 +1234,6 @@ QString findDesktopFile(const QString& dirName, const QString& desktopName)
 }
 
 
-/************************************************
-
- ************************************************/
 QString findDesktopFile(const QString& desktopName)
 {
     QStringList dataDirs = XdgDirs::dataDirs();
@@ -1364,11 +1250,6 @@ QString findDesktopFile(const QString& desktopName)
 }
 
 
-
-
-/************************************************
-
- ************************************************/
 XdgDesktopFile* XdgDesktopFileCache::getFile(const QString& fileName)
 {
     if (instance().m_fileCache.contains(fileName))
@@ -1432,7 +1313,6 @@ XdgDesktopFileCache & XdgDesktopFileCache::instance()
 }
 
 
-
 /*! 
  * Handles files with a syntax similar to desktopfiles as QSettings files.
  * The differences between ini-files and desktopfiles are:
@@ -1487,6 +1367,7 @@ bool readDesktopFile(QIODevice & device, QSettings::SettingsMap & map)
     return true;
 }
 
+
 /*! See readDesktopFile
  */
 bool writeDesktopFile(QIODevice & device, const QSettings::SettingsMap & map)
@@ -1530,9 +1411,6 @@ bool writeDesktopFile(QIODevice & device, const QSettings::SettingsMap & map)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgDesktopFileCache::initialize(const QString& dirName)
 {
     QDir dir(dirName);
@@ -1580,6 +1458,7 @@ void XdgDesktopFileCache::initialize(const QString& dirName)
 
 }
 
+
 XdgDesktopFile* XdgDesktopFileCache::load(const QString& fileName)
 {
     XdgDesktopFile* desktopFile = new XdgDesktopFile();
@@ -1587,6 +1466,7 @@ XdgDesktopFile* XdgDesktopFileCache::load(const QString& fileName)
     return desktopFile;
 }
 
+
 void loadMimeCacheDir(const QString& dirName, QHash<QString, QList<XdgDesktopFile*> > & cache)
 {
     QDir dir(dirName);
@@ -1649,12 +1529,12 @@ XdgDesktopFileCache::XdgDesktopFileCache() :
 {
 }
 
+
 XdgDesktopFileCache::~XdgDesktopFileCache()
 {
 }
 
 
-
 void XdgDesktopFileCache::initialize()
 {
     QStringList dataDirs = XdgDirs::dataDirs();
@@ -1686,9 +1566,6 @@ QList<XdgDesktopFile*>  XdgDesktopFileCache::getApps(const QString& mimetype)
 }
 
 
-/************************************************
-
- ************************************************/
 XdgDesktopFile* XdgDesktopFileCache::getDefaultApp(const QString& mimetype)
 {
     // First, we look in ~/.local/share/applications/mimeapps.list, /usr/local/share/applications/mimeapps.list and
@@ -1733,4 +1610,3 @@ XdgDesktopFile* XdgDesktopFileCache::getDefaultApp(const QString& mimetype)
     XdgDesktopFile* desktopFile = apps.isEmpty() ? 0 : apps[0];
     return desktopFile;
 }
-
diff --git a/xdgdesktopfile.h b/xdgdesktopfile.h
index 0712ca6..861eba3 100644
--- a/xdgdesktopfile.h
+++ b/xdgdesktopfile.h
@@ -33,7 +33,6 @@
 #include "xdgmacros.h"
 
 #include <QSharedDataPointer>
-//#include <QObject>
 #include <QString>
 #include <QVariant>
 #include <QStringList>
diff --git a/xdgdirs.cpp b/xdgdirs.cpp
index 32c178b..b80aad8 100644
--- a/xdgdirs.cpp
+++ b/xdgdirs.cpp
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #include "xdgdirs.h"
 #include <stdlib.h>
 #include <QDir>
@@ -62,6 +61,7 @@ void fixBashShortcuts(QString &s)
         s = QString(getenv("HOME")) + (s).mid(1);
 }
 
+
 void removeEndingSlash(QString &s)
 {
     // We don't check for empty strings. Caller must check it.
@@ -71,6 +71,7 @@ void removeEndingSlash(QString &s)
         s.chop(1);
 }
 
+
 QString createDirectory(const QString &dir)
 {
     QDir d(dir);
@@ -86,6 +87,7 @@ QString createDirectory(const QString &dir)
     return r;
 }
 
+
 void cleanAndAddPostfix(QStringList &dirs, const QString& postfix)
 {
     const int N = dirs.count();
@@ -97,6 +99,7 @@ void cleanAndAddPostfix(QStringList &dirs, const QString& postfix)
     }
 }
 
+
 QString XdgDirs::userDir(XdgDirs::UserDirectory dir)
 {
     // possible values for UserDirectory
@@ -144,9 +147,6 @@ QString XdgDirs::userDir(XdgDirs::UserDirectory dir)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgDirs::setUserDir(XdgDirs::UserDirectory dir, const QString& value, bool createDir)
 {
     // possible values for UserDirectory
@@ -207,9 +207,6 @@ bool XdgDirs::setUserDir(XdgDirs::UserDirectory dir, const QString& value, bool
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDirs::dataHome(bool createDir)
 {
     QString s = QStandardPaths::writableLocation(QStandardPaths::DataLocation);
@@ -222,9 +219,6 @@ QString XdgDirs::dataHome(bool createDir)
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDirs::configHome(bool createDir)
 {
     QString s = QStandardPaths::writableLocation(QStandardPaths::GenericConfigLocation);
@@ -237,9 +231,6 @@ QString XdgDirs::configHome(bool createDir)
 }
 
 
-/************************************************
-
- ************************************************/
 QStringList XdgDirs::dataDirs(const QString &postfix)
 {
     QString d = QFile::decodeName(qgetenv("XDG_DATA_DIRS"));
@@ -259,9 +250,6 @@ QStringList XdgDirs::dataDirs(const QString &postfix)
 }
 
 
-/************************************************
-
- ************************************************/
 QStringList XdgDirs::configDirs(const QString &postfix)
 {
     QStringList dirs;
@@ -276,9 +264,6 @@ QStringList XdgDirs::configDirs(const QString &postfix)
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDirs::cacheHome(bool createDir)
 {
     QString s = QStandardPaths::writableLocation(QStandardPaths::GenericCacheLocation);
@@ -291,9 +276,6 @@ QString XdgDirs::cacheHome(bool createDir)
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDirs::runtimeDir()
 {
     QString result = QStandardPaths::writableLocation(QStandardPaths::RuntimeLocation);
@@ -303,9 +285,6 @@ QString XdgDirs::runtimeDir()
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgDirs::autostartHome(bool createDir)
 {
     QString s = QString("%1/autostart").arg(configHome(createDir));
@@ -321,9 +300,6 @@ QString XdgDirs::autostartHome(bool createDir)
 }
 
 
-/************************************************
-
- ************************************************/
 QStringList XdgDirs::autostartDirs(const QString &postfix)
 {
     QStringList dirs;
diff --git a/xdgdirs.h b/xdgdirs.h
index 0ac1298..fb333d2 100644
--- a/xdgdirs.h
+++ b/xdgdirs.h
@@ -25,8 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
-
 #ifndef QTXDG_XDGDIRS_H
 #define QTXDG_XDGDIRS_H
 
diff --git a/xdgicon.cpp b/xdgicon.cpp
index f92886b..231a3d5 100644
--- a/xdgicon.cpp
+++ b/xdgicon.cpp
@@ -25,8 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
-
 #include "xdgicon.h"
 
 #include <QString>
@@ -40,9 +38,6 @@
 
 #define DEFAULT_APP_ICON "application-x-executable"
 
-/************************************************
-
- ************************************************/
 static void qt_cleanup_icon_cache();
 typedef QCache<QString, QIcon> IconCache;
 
@@ -63,18 +58,11 @@ static void qt_cleanup_icon_cache()
 }
 
 
-
-/************************************************
-
- ************************************************/
 XdgIcon::XdgIcon()
 {
 }
 
 
-/************************************************
-
- ************************************************/
 XdgIcon::~XdgIcon()
 {
 }
@@ -159,9 +147,6 @@ QIcon XdgIcon::fromTheme(const QStringList& iconNames, const QIcon& fallback)
 }
 
 
-/************************************************
-
- ************************************************/
 QIcon XdgIcon::fromTheme(const QString &iconName,
                          const QString &fallbackIcon1,
                          const QString &fallbackIcon2,
@@ -178,18 +163,13 @@ QIcon XdgIcon::fromTheme(const QString &iconName,
     return fromTheme(icons);
 }
 
-/************************************************
 
- ************************************************/
 QIcon XdgIcon::defaultApplicationIcon()
 {
     return fromTheme(DEFAULT_APP_ICON);
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgIcon::defaultApplicationIconName()
 {
     return DEFAULT_APP_ICON;
diff --git a/xdgicon.h b/xdgicon.h
index 63fe6ea..594415e 100644
--- a/xdgicon.h
+++ b/xdgicon.h
@@ -25,8 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
-
 #ifndef QTXDG_XDGICON_H
 #define QTXDG_XDGICON_H
 
diff --git a/xdgmenu.cpp b/xdgmenu.cpp
index 9e538e8..472d0fd 100644
--- a/xdgmenu.cpp
+++ b/xdgmenu.cpp
@@ -26,7 +26,6 @@
  * END_COMMON_COPYRIGHT_HEADER */
 
 
-
 #include "xdgmenu.h"
 #include "xdgmenu_p.h"
 #include "xdgmenureader.h"
@@ -71,9 +70,6 @@ void installTranslation(const QString &name)
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenu::XdgMenu(QObject *parent) :
     QObject(parent),
     d_ptr(new XdgMenuPrivate(this))
@@ -82,9 +78,6 @@ XdgMenu::XdgMenu(QObject *parent) :
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenu::~XdgMenu()
 {
     Q_D(XdgMenu);
@@ -92,9 +85,6 @@ XdgMenu::~XdgMenu()
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuPrivate::XdgMenuPrivate(XdgMenu *parent):
     mOutDated(true),
     q_ptr(parent)
@@ -111,9 +101,6 @@ XdgMenuPrivate::XdgMenuPrivate(XdgMenu *parent):
 }
 
 
-/************************************************
-
- ************************************************/
 const QString XdgMenu::logDir() const
 {
     Q_D(const XdgMenu);
@@ -121,9 +108,6 @@ const QString XdgMenu::logDir() const
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenu::setLogDir(const QString& directory)
 {
     Q_D(XdgMenu);
@@ -131,9 +115,6 @@ void XdgMenu::setLogDir(const QString& directory)
 }
 
 
-/************************************************
-
- ************************************************/
 const QDomDocument XdgMenu::xml() const
 {
     Q_D(const XdgMenu);
@@ -141,9 +122,6 @@ const QDomDocument XdgMenu::xml() const
 }
 
 
-/************************************************
-
- ************************************************/
 QString XdgMenu::menuFileName() const
 {
     Q_D(const XdgMenu);
@@ -151,28 +129,26 @@ QString XdgMenu::menuFileName() const
 }
 
 
-/************************************************
-
- ************************************************/
 QStringList XdgMenu::environments()
 {
     Q_D(XdgMenu);
     return d->mEnvironments;
 }
 
+
 void XdgMenu::setEnvironments(const QStringList &envs)
 {
     Q_D(XdgMenu);
     d->mEnvironments = envs;
 }
+
+
 void XdgMenu::setEnvironments(const QString &env)
 {
     setEnvironments(QStringList() << env);
 }
 
-/************************************************
 
- ************************************************/
 const QString XdgMenu::errorString() const
 {
     Q_D(const XdgMenu);
@@ -180,9 +156,6 @@ const QString XdgMenu::errorString() const
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenu::read(const QString& menuFileName)
 {
     Q_D(XdgMenu);
@@ -241,9 +214,6 @@ bool XdgMenu::read(const QString& menuFileName)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenu::save(const QString& fileName)
 {
     Q_D(const XdgMenu);
@@ -279,9 +249,6 @@ void XdgMenuPrivate::load(const QString& fileName)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::saveLog(const QString& logFileName)
 {
     Q_Q(XdgMenu);
@@ -290,9 +257,6 @@ void XdgMenuPrivate::saveLog(const QString& logFileName)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::mergeMenus(QDomElement& element)
 {
     QHash<QString, QDomElement> menus;
@@ -333,9 +297,6 @@ void XdgMenuPrivate::mergeMenus(QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::simplify(QDomElement& element)
 {
     MutableDomElementIterator it(element);
@@ -393,9 +354,6 @@ void XdgMenuPrivate::simplify(QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::prependChilds(QDomElement& srcElement, QDomElement& destElement)
 {
     MutableDomElementIterator it(srcElement);
@@ -419,9 +377,6 @@ void XdgMenuPrivate::prependChilds(QDomElement& srcElement, QDomElement& destEle
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::appendChilds(QDomElement& srcElement, QDomElement& destElement)
 {
     MutableDomElementIterator it(srcElement);
@@ -487,9 +442,6 @@ QDomElement XdgMenu::findMenu(QDomElement& baseElement, const QString& path, boo
 }
 
 
-/************************************************
-
- ************************************************/
 bool isParent(const QDomElement& parent, const QDomElement& child)
 {
     QDomNode  n = child;
@@ -574,9 +526,6 @@ void XdgMenuPrivate::deleteDeletedMenus(QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::processDirectoryEntries(QDomElement& element, const QStringList& parentDirs)
 {
     QStringList dirs;
@@ -631,9 +580,6 @@ void XdgMenuPrivate::processDirectoryEntries(QDomElement& element, const QString
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuPrivate::loadDirectoryFile(const QString& fileName, QDomElement& element)
 {
     XdgDesktopFile file;
@@ -653,9 +599,6 @@ bool XdgMenuPrivate::loadDirectoryFile(const QString& fileName, QDomElement& ele
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::processApps(QDomElement& element)
 {
     Q_Q(XdgMenu);
@@ -664,9 +607,6 @@ void XdgMenuPrivate::processApps(QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::deleteEmpty(QDomElement& element)
 {
     MutableDomElementIterator it(element, "Menu");
@@ -686,9 +626,6 @@ void XdgMenuPrivate::deleteEmpty(QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::processLayouts(QDomElement& element)
 {
     XdgMenuLayoutProcessor proc(element);
@@ -696,9 +633,6 @@ void XdgMenuPrivate::processLayouts(QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::fixSeparators(QDomElement& element)
 {
 
@@ -768,9 +702,6 @@ QString XdgMenu::getMenuFileName(const QString& baseName)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenu::addWatchPath(const QString &path)
 {
     Q_D(XdgMenu);
@@ -785,9 +716,6 @@ void XdgMenu::addWatchPath(const QString &path)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenu::isOutDated() const
 {
     Q_D(const XdgMenu);
@@ -795,9 +723,6 @@ bool XdgMenu::isOutDated() const
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::rebuild()
 {
     Q_Q(XdgMenu);
@@ -812,9 +737,6 @@ void XdgMenuPrivate::rebuild()
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuPrivate::clearWatcher()
 {
     QStringList sl;
diff --git a/xdgmenu.h b/xdgmenu.h
index 54357ea..a88aa64 100644
--- a/xdgmenu.h
+++ b/xdgmenu.h
@@ -25,8 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
-
 #ifndef QTXDG_XDGMENU_H
 #define QTXDG_XDGMENU_H
 
@@ -36,7 +34,6 @@
 #include <QStringList>
 #include <QtXml/QDomDocument>
 
-
 class QDomDocument;
 class QDomElement;
 class XdgMenuPrivate;
diff --git a/xdgmenu_p.h b/xdgmenu_p.h
index fb2c936..34443aa 100644
--- a/xdgmenu_p.h
+++ b/xdgmenu_p.h
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #include "xdgmenu.h"
 #include <QObject>
 #include <QFileSystemWatcher>
@@ -85,4 +84,3 @@ private:
     XdgMenu* const q_ptr;
     Q_DECLARE_PUBLIC(XdgMenu)
 };
-
diff --git a/xdgmenuapplinkprocessor.cpp b/xdgmenuapplinkprocessor.cpp
index c7ceedc..02265d6 100644
--- a/xdgmenuapplinkprocessor.cpp
+++ b/xdgmenuapplinkprocessor.cpp
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #include "xdgmenu.h"
 #include "xdgmenuapplinkprocessor.h"
 #include "xmlhelper.h"
@@ -34,9 +33,6 @@
 #include <QDir>
 
 
-/************************************************
-
- ************************************************/
 XdgMenuApplinkProcessor::XdgMenuApplinkProcessor(QDomElement& element,  XdgMenu* menu, XdgMenuApplinkProcessor *parent) :
     QObject(parent)
 {
@@ -56,17 +52,11 @@ XdgMenuApplinkProcessor::XdgMenuApplinkProcessor(QDomElement& element,  XdgMenu*
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuApplinkProcessor::~XdgMenuApplinkProcessor()
 {
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuApplinkProcessor::run()
 {
     step1();
@@ -74,9 +64,6 @@ void XdgMenuApplinkProcessor::run()
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuApplinkProcessor::step1()
 {
     fillAppFileInfoList();
@@ -108,9 +95,6 @@ void XdgMenuApplinkProcessor::step1()
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuApplinkProcessor::step2()
 {
     // Create AppLinks elements ...........................
@@ -199,9 +183,6 @@ void XdgMenuApplinkProcessor::fillAppFileInfoList()
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuApplinkProcessor::findDesktopFiles(const QString& dirName, const QString& prefix)
 {
     QDir dir(dirName);
@@ -229,9 +210,6 @@ void XdgMenuApplinkProcessor::findDesktopFiles(const QString& dirName, const QSt
 }
 
 
-/************************************************
- Create rules
- ************************************************/
 void XdgMenuApplinkProcessor::createRules()
 {
     MutableDomElementIterator i(mElement, QString());
diff --git a/xdgmenuapplinkprocessor.h b/xdgmenuapplinkprocessor.h
index 6b85331..0bd0cb5 100644
--- a/xdgmenuapplinkprocessor.h
+++ b/xdgmenuapplinkprocessor.h
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #ifndef QTXDG_XDGMENUAPPLINKPROCESSOR_H
 #define QTXDG_XDGMENUAPPLINKPROCESSOR_H
 
@@ -45,6 +44,7 @@ typedef QLinkedList<XdgMenuAppFileInfo*> XdgMenuAppFileInfoList;
 typedef QHash<QString, XdgMenuAppFileInfo*> XdgMenuAppFileInfoHash;
 typedef QHashIterator<QString, XdgMenuAppFileInfo*> XdgMenuAppFileInfoHashIterator;
 
+
 class XdgMenuApplinkProcessor : public QObject
 {
     Q_OBJECT
@@ -98,5 +98,4 @@ private:
     QString mId;
 };
 
-
 #endif // QTXDG_XDGMENUAPPLINKPROCESSOR_H
diff --git a/xdgmenulayoutprocessor.cpp b/xdgmenulayoutprocessor.cpp
index f4aa414..75089c2 100644
--- a/xdgmenulayoutprocessor.cpp
+++ b/xdgmenulayoutprocessor.cpp
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #include "xdgmenulayoutprocessor.h"
 #include "xmlhelper.h"
 #include <QDebug>
@@ -36,9 +35,7 @@
 QDomElement findLastElementByTag(const QDomElement element, const QString tagName);
 int childsCount(const QDomElement& element);
 
-/************************************************
 
- ************************************************/
 QDomElement findLastElementByTag(const QDomElement element, const QString tagName)
 {
     QDomNodeList l = element.elementsByTagName(tagName);
@@ -100,9 +97,6 @@ XdgMenuLayoutProcessor::XdgMenuLayoutProcessor(QDomElement& element):
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuLayoutProcessor::XdgMenuLayoutProcessor(QDomElement& element, XdgMenuLayoutProcessor *parent):
     mElement(element)
 {
@@ -127,9 +121,6 @@ XdgMenuLayoutProcessor::XdgMenuLayoutProcessor(QDomElement& element, XdgMenuLayo
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuLayoutProcessor::setParams(QDomElement defaultLayout, LayoutParams *result)
 {
     if (defaultLayout.hasAttribute("show_empty"))
@@ -149,9 +140,6 @@ void XdgMenuLayoutProcessor::setParams(QDomElement defaultLayout, LayoutParams *
 }
 
 
-/************************************************
-
- ************************************************/
 QDomElement XdgMenuLayoutProcessor::searchElement(const QString &tagName, const QString &attributeName, const QString &attributeValue) const
 {
     DomElementIterator it(mElement, tagName);
@@ -168,9 +156,6 @@ QDomElement XdgMenuLayoutProcessor::searchElement(const QString &tagName, const
 }
 
 
-/************************************************
-
- ************************************************/
 int childsCount(const QDomElement& element)
 {
     int count = 0;
@@ -186,9 +171,6 @@ int childsCount(const QDomElement& element)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuLayoutProcessor::run()
 {
     QDomDocument doc = mLayout.ownerDocument();
@@ -427,4 +409,3 @@ void XdgMenuLayoutProcessor::processMergeTag(const QDomElement &element)
 
     mResult.removeChild(element);
 }
-
diff --git a/xdgmenulayoutprocessor.h b/xdgmenulayoutprocessor.h
index d67c61b..c07dfd6 100644
--- a/xdgmenulayoutprocessor.h
+++ b/xdgmenulayoutprocessor.h
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #ifndef QTXDG_XDGMENULAYOUTPROCESSOR_H
 #define QTXDG_XDGMENULAYOUTPROCESSOR_H
 
diff --git a/xdgmenureader.cpp b/xdgmenureader.cpp
index 9e6b14b..6207eb9 100644
--- a/xdgmenureader.cpp
+++ b/xdgmenureader.cpp
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #include "xdgmenureader.h"
 #include "xdgmenu.h"
 #include "xdgdirs.h"
@@ -41,10 +40,6 @@
 #include <QtXml/QDomNode>
 
 
-
-/************************************************
-
- ************************************************/
 XdgMenuReader::XdgMenuReader(XdgMenu* menu, XdgMenuReader*  parentReader, QObject *parent) :
     QObject(parent),
     mMenu(menu)
@@ -55,18 +50,12 @@ XdgMenuReader::XdgMenuReader(XdgMenu* menu, XdgMenuReader*  parentReader, QObjec
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuReader::~XdgMenuReader()
 {
 
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuReader::load(const QString& fileName, const QString& baseDir)
 {
     if (fileName.isEmpty())
@@ -433,9 +422,6 @@ void XdgMenuReader::mergeFile(const QString& fileName, QDomElement& element, QSt
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuReader::mergeDir(const QString& dirName, QDomElement& element, QStringList* mergedFiles)
 {
     QFileInfo dirInfo(mDirName, dirName);
@@ -450,7 +436,3 @@ void XdgMenuReader::mergeDir(const QString& dirName, QDomElement& element, QStri
             mergeFile(file.canonicalFilePath(), element, mergedFiles);
     }
 }
-
-
-
-
diff --git a/xdgmenureader.h b/xdgmenureader.h
index 1962cfb..27e1f5e 100644
--- a/xdgmenureader.h
+++ b/xdgmenureader.h
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #ifndef QTXDG_XDGMENUREADER_H
 #define QTXDG_XDGMENUREADER_H
 
@@ -34,6 +33,7 @@
 #include <QStringList>
 #include <QtXml/QDomDocument>
 #include <QtXml/QDomElement>
+
 class XdgMenu;
 class XdgMenuReader : public QObject
 {
diff --git a/xdgmenurules.cpp b/xdgmenurules.cpp
index eca7dd1..eab1da7 100644
--- a/xdgmenurules.cpp
+++ b/xdgmenurules.cpp
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 /*********************************************************************
   See: http://standards.freedesktop.org/desktop-entry-spec
 
@@ -38,10 +37,6 @@
 #include <QDebug>
 
 
-
-/************************************************
-
- ************************************************/
 XdgMenuRule::XdgMenuRule(const QDomElement& element, QObject* parent) :
     QObject(parent)
 {
@@ -49,17 +44,11 @@ XdgMenuRule::XdgMenuRule(const QDomElement& element, QObject* parent) :
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuRule::~XdgMenuRule()
 {
 }
 
 
-
-
-
 /************************************************
  The <Or> element contains a list of matching rules. If any of the matching rules
  inside the <Or> element match a desktop entry, then the entire <Or> rule matches
@@ -100,9 +89,6 @@ XdgMenuRuleOr::XdgMenuRuleOr(const QDomElement& element, QObject* parent) :
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRuleOr::check(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     for (QLinkedList<XdgMenuRule*>::Iterator i=mChilds.begin(); i!=mChilds.end(); ++i)
@@ -124,9 +110,6 @@ XdgMenuRuleAnd::XdgMenuRuleAnd(const QDomElement& element, QObject *parent) :
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRuleAnd::check(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     for (QLinkedList<XdgMenuRule*>::Iterator i=mChilds.begin(); i!=mChilds.end(); ++i)
@@ -136,7 +119,6 @@ bool XdgMenuRuleAnd::check(const QString& desktopFileId, const XdgDesktopFile& d
 }
 
 
-
 /************************************************
  The <Not> element contains a list of matching rules. If any of the matching rules
  inside the <Not> element matches a desktop entry, then the entire <Not> rule does
@@ -150,16 +132,12 @@ XdgMenuRuleNot::XdgMenuRuleNot(const QDomElement& element, QObject *parent) :
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRuleNot::check(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     return ! XdgMenuRuleOr::check(desktopFileId, desktopFile);
 }
 
 
-
 /************************************************
  The <Filename> element is the most basic matching rule. It matches a desktop entry
  if the desktop entry has the given desktop-file id. See Desktop-File Id.
@@ -172,9 +150,6 @@ XdgMenuRuleFileName::XdgMenuRuleFileName(const QDomElement& element, QObject *pa
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRuleFileName::check(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     Q_UNUSED(desktopFile)
@@ -183,7 +158,6 @@ bool XdgMenuRuleFileName::check(const QString& desktopFileId, const XdgDesktopFi
 }
 
 
-
 /************************************************
  The <Category> element is another basic matching predicate. It matches a desktop entry
  if the desktop entry has the given category in its Categories field.
@@ -195,9 +169,6 @@ XdgMenuRuleCategory::XdgMenuRuleCategory(const QDomElement& element, QObject *pa
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRuleCategory::check(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     Q_UNUSED(desktopFileId)
@@ -215,9 +186,6 @@ XdgMenuRuleAll::XdgMenuRuleAll(const QDomElement& element, QObject *parent) :
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRuleAll::check(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     Q_UNUSED(desktopFileId)
@@ -226,46 +194,29 @@ bool XdgMenuRuleAll::check(const QString& desktopFileId, const XdgDesktopFile& d
 }
 
 
-
-
-/************************************************
-
- ************************************************/
 XdgMenuRules::XdgMenuRules(QObject* parent) :
     QObject(parent)
 {
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuRules::~XdgMenuRules()
 {
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuRules::addInclude(const QDomElement& element)
 {
     mIncludeRules.append(new XdgMenuRuleOr(element, this));
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuRules::addExclude(const QDomElement& element)
 {
     mExcludeRules.append(new XdgMenuRuleOr(element, this));
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRules::checkInclude(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     for (QLinkedList<XdgMenuRule*>::Iterator i=mIncludeRules.begin(); i!=mIncludeRules.end(); ++i)
@@ -275,9 +226,6 @@ bool XdgMenuRules::checkInclude(const QString& desktopFileId, const XdgDesktopFi
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuRules::checkExclude(const QString& desktopFileId, const XdgDesktopFile& desktopFile)
 {
     for (QLinkedList<XdgMenuRule*>::Iterator i=mExcludeRules.begin(); i!=mExcludeRules.end(); ++i)
@@ -285,5 +233,3 @@ bool XdgMenuRules::checkExclude(const QString& desktopFileId, const XdgDesktopFi
 
     return false;
 }
-
-
diff --git a/xdgmenurules.h b/xdgmenurules.h
index 9ce558f..422a33f 100644
--- a/xdgmenurules.h
+++ b/xdgmenurules.h
@@ -40,6 +40,7 @@
 
 #include "xdgdesktopfile.h"
 
+
 class XdgMenuRule : public QObject
 {
     Q_OBJECT
diff --git a/xdgmenuwidget.cpp b/xdgmenuwidget.cpp
index 8225c24..d2e6c79 100644
--- a/xdgmenuwidget.cpp
+++ b/xdgmenuwidget.cpp
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #include "xdgmenuwidget.h"
 #include "xdgicon.h"
 #include "xmlhelper.h"
@@ -40,6 +39,7 @@
 #include <QtGui/QMouseEvent>
 #include <QApplication>
 
+
 class XdgMenuWidgetPrivate
 {
 private:
@@ -66,10 +66,6 @@ private:
 };
 
 
-
-/************************************************
-
- ************************************************/
 XdgMenuWidget::XdgMenuWidget(const XdgMenu& xdgMenu, const QString& title, QWidget* parent):
     QMenu(parent),
     d_ptr(new XdgMenuWidgetPrivate(this))
@@ -78,9 +74,7 @@ XdgMenuWidget::XdgMenuWidget(const XdgMenu& xdgMenu, const QString& title, QWidg
     setTitle(XdgMenuWidgetPrivate::escape(title));
 }
 
-/************************************************
 
- ************************************************/
 XdgMenuWidget::XdgMenuWidget(const QDomElement& menuElement, QWidget* parent):
     QMenu(parent),
     d_ptr(new XdgMenuWidgetPrivate(this))
@@ -89,9 +83,6 @@ XdgMenuWidget::XdgMenuWidget(const QDomElement& menuElement, QWidget* parent):
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuWidget::XdgMenuWidget(const XdgMenuWidget& other, QWidget* parent):
     QMenu(parent),
     d_ptr(new XdgMenuWidgetPrivate(this))
@@ -100,9 +91,6 @@ XdgMenuWidget::XdgMenuWidget(const XdgMenuWidget& other, QWidget* parent):
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuWidgetPrivate::init(const QDomElement& xml)
 {
     Q_Q(XdgMenuWidget);
@@ -131,18 +119,12 @@ void XdgMenuWidgetPrivate::init(const QDomElement& xml)
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuWidget::~XdgMenuWidget()
 {
     delete d_ptr;
 }
 
 
-/************************************************
-
- ************************************************/
 XdgMenuWidget& XdgMenuWidget::operator=(const XdgMenuWidget& other)
 {
     Q_D(XdgMenuWidget);
@@ -152,9 +134,6 @@ XdgMenuWidget& XdgMenuWidget::operator=(const XdgMenuWidget& other)
 }
 
 
-/************************************************
-
- ************************************************/
 bool XdgMenuWidget::event(QEvent* event)
 {
     Q_D(XdgMenuWidget);
@@ -176,9 +155,6 @@ bool XdgMenuWidget::event(QEvent* event)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuWidgetPrivate::mouseMoveEvent(QMouseEvent *event)
 {
     if (!(event->buttons() & Qt::LeftButton))
@@ -204,9 +180,6 @@ void XdgMenuWidgetPrivate::mouseMoveEvent(QMouseEvent *event)
 }
 
 
-/************************************************
-
- ************************************************/
 void XdgMenuWidgetPrivate::buildMenu()
 {
     Q_Q(XdgMenuWidget);
@@ -237,9 +210,6 @@ void XdgMenuWidgetPrivate::buildMenu()
 }
 
 
-/************************************************
-
- ************************************************/
 XdgAction* XdgMenuWidgetPrivate::createAction(const QDomElement& xml)
 {
     Q_Q(XdgMenuWidget);
@@ -269,4 +239,3 @@ QString XdgMenuWidgetPrivate::escape(QString string)
 {
     return string.replace("&", "&&");
 }
-
diff --git a/xdgmenuwidget.h b/xdgmenuwidget.h
index 53e24e6..4b73412 100644
--- a/xdgmenuwidget.h
+++ b/xdgmenuwidget.h
@@ -25,7 +25,6 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-
 #ifndef QTXDG_XDGMENUWIDGET_H
 #define QTXDG_XDGMENUWIDGET_H
 
diff --git a/xdgmimetype.cpp b/xdgmimetype.cpp
index 7059e5c..7bb8987 100644
--- a/xdgmimetype.cpp
+++ b/xdgmimetype.cpp
@@ -22,6 +22,7 @@
 
 #include "xdgicon.h"
 
+
 class XdgMimeTypePrivate : public QSharedData {
 public:
     XdgMimeTypePrivate();
@@ -33,35 +34,41 @@ public:
     bool computed;
 };
 
+
 XdgMimeTypePrivate::XdgMimeTypePrivate()
     : computed(false)
 {
 }
 
+
 XdgMimeTypePrivate::XdgMimeTypePrivate(const XdgMimeType& other)
     : iconName(other.dx->iconName),
       computed(other.dx->computed)
 {
 }
 
+
 XdgMimeType::XdgMimeType()
     : QMimeType(),
       dx(new XdgMimeTypePrivate())
 {
 }
 
+
 XdgMimeType::XdgMimeType(const QMimeType& mime)
     : QMimeType(mime),
       dx(new XdgMimeTypePrivate())
 {
 }
 
+
 XdgMimeType::XdgMimeType(const XdgMimeType& mime)
     : QMimeType(mime),
       dx(mime.dx)
 {
 }
 
+
 XdgMimeType &XdgMimeType::operator=(const XdgMimeType &other)
 {
     QMimeType::operator =(other);
@@ -72,10 +79,12 @@ XdgMimeType &XdgMimeType::operator=(const XdgMimeType &other)
     return *this;
 }
 
+
 XdgMimeType::~XdgMimeType()
 {
 }
 
+
 QString XdgMimeType::iconName() const
 {
     if (dx->computed) {
@@ -98,6 +107,7 @@ QString XdgMimeType::iconName() const
     }
 }
 
+
 QIcon XdgMimeType::icon() const
 {
     return XdgIcon::fromTheme((iconName()));
-- 
1.9.0

